packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.form.story.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable react/prop-types, react/display-name */
```

Comment:
I'm wondering if I should add additional vrts with resetting the `LocalizedRichTextInput` as part of this fix PR?

Reply:
If you want, it would be nice yes.